pkg/syncer/sync.go

```diff
@@ -106,6 +92,17 @@ func (s *Syncer) SyncPendingCharts(names ...string) error {
 			}
 		}
 
+		// Read final chart metadata
+		configFilePath := fmt.Sprintf("%s/Chart.yaml", chartPath)
+		chartConfig, err := ioutil.ReadFile(configFilePath)
+		if err != nil {
+			return err
+		}
+		metadata := &helmchart.Metadata{}
+		if err = yaml.Unmarshal(chartConfig, metadata); err != nil {
+			return err
+		}
```

Comment:
note that we are in a loop, let's use `multierror` and continue

```suggestion
		// Read final chart metadata
		configFilePath := fmt.Sprintf("%s/Chart.yaml", chartPath)
		chartConfig, err := ioutil.ReadFile(configFilePath)
		if err != nil {
			klog.Errorf("unable to read %q metadata: %+v", id, err)
			errs = multierror.Append(errs, errors.Trace(err))
			continue
		}
		metadata := &helmchart.Metadata{}
		if err = yaml.Unmarshal(chartConfig, metadata); err != nil {
			klog.Errorf("unable to decode %q metadata: %+v", id, err)
			errs = multierror.Append(errs, errors.Trace(err))
			continue
		}
```